### PR TITLE
updated docs to include support for GitHub only & added RELEASE_NAME …

### PR DESCRIPTION
**Pull request by abhijitkini**
…env variable to the list

### CODE REVIEW

**Comment by abhijitkini** [comment]
https://github.com/Shippable/docsv2/issues/347

**Comment by abhijitkini** [comment]
https://github.com/Shippable/support/issues/2615

**Comment by abhijitkini** [comment]
https://github.com/Shippable/docsv2/issues/349

File: sources/ci_overview.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Shippable automatically builds and tests your repositories when -
 
 You can also run a manual build through the UI clicking on the `Build` button for any project or branch.
 
-Please note that we do not automatically trigger builds if you push a tag. You can still run a manual build after pushing the tag. 
+Please note that we do not automatically trigger builds if you push a tag. You can still run a manual build after pushing the tag. Go [here](ci_projects.md#configure-webhook-events-for-triggering-builds) for details on configuring these settings.
```
Comment (manishas) [inline sources/ci_overview.md]: we should say - 

By default, we do not automatically trigger builds if you push a tag or create a release.

**Pull request merged**